packages/nimble-components/src/table-column/mapping/tests/table-column-mapping.spec.ts

```diff
@@ -27,22 +28,18 @@ interface SimpleTableRecord extends TableRecord {
     field2?: MappingKey | null;
 }
 
-interface BasicIconMapping {
+interface BaseMapping {
     key?: MappingKey;
     text?: string;
-    icon?: string;
-    textHidden?: boolean;
 }
 
-interface BasicSpinnerMapping {
-    key?: MappingKey;
-    text?: string;
+interface IconMapping extends BaseMapping {
+    icon?: string;
     textHidden?: boolean;
 }
 
-interface BasicTextMapping {
-    key?: MappingKey;
-    text?: string;
+interface SpinnerMapping extends BaseMapping {
+    textHidden?: boolean;
 }
 
```

Comment:
Can we postfix `Test` or something to these, `BaseMappingTest` `IconMappingTest` etc. I think just standalone it was a little confusing looking at the template.

Reply:
I've updated to add `Test` as a prefix since, at least to me, it makes more sense to call these "test mappings" rather than "mapping tests".